src/base/projects/BranchSelector.spec.ts

```diff
@@ -141,26 +140,3 @@ describe("Layout", () => {
     cy.get("div.hash.desktop").should("be.visible");
   });
 });
-
```

Comment:
Couldn't get the tests to run locally to debug this: https://github.com/radicle-dev/radicle-interface/issues/318.